config/prow/plugins.yaml

```diff
@@ -247,6 +247,7 @@ blockades:
 - repos:
   - kubernetes/kubernetes
   blockregexps:
+  
```

Comment:
Why is this new line here?

Reply:
Once the new line is removed, this PR should be ready
I verified that the 1.28 milestone is present for k/website and the `dev-1.28` branch is created in k/website

Reply:
Removed the new line. It looks like it was fat fingered.